code/modules/projectiles/guns/shotguns.dm

```diff
@@ -1025,11 +1025,53 @@ can cause issues with ammo types getting mixed up during the burst.
 	return ..()
 
 //-------------------------------------------------------
+
+/obj/item/weapon/gun/shotgun/pump/dual_tube
+	name = "\improper generic dual-tube pump shotgun"
```

Comment:
do we need the improper? given its lowercase anyway

Reply:
nope